cmd/paratime/statistics.go

```diff
@@ -69,24 +66,19 @@ type entityStats struct {
 	// Rounds entity node was a proposer.
 	roundsProposer uint64
 
-	// How many times entity node proposed a timeout.
-	proposedTimeout uint64
-
 	// How many good blocks committed while being primary worker.
-	committeedGoodBlocksPrimary uint64
+	committedGoodBlocksPrimary uint64
 	// How many bad blocs committed while being primary worker.
-	committeedBadBlocksPrimary uint64
+	committedBadBlocksPrimary uint64
 	// How many good blocks committed while being backup worker.
-	committeedGoodBlocksBackup uint64
+	committedGoodBlocksBackup uint64
 	// How many bad blocks committed while being backup worker.
-	committeedBadBlocksBackup uint64
+	committedBadBlocksBackup uint64
 
 	// How many rounds missed committing a block while being a primary worker.
 	missedPrimary uint64
 	// How many rounds missed committing a block while being a backup worker (and discrepancy detection was invoked).
 	missedBackup uint64
-	// How many rounds proposer timeout was triggered while being the proposer.
-	missedProposer uint64
```

Comment:
Is this something not relevant in 23.0 anymore?

Reply:
That's right, the way backup proposers work has been totally changed. See https://github.com/oasisprotocol/oasis-core/pull/5354 for details.